arch/arm64/boot/dts/qcom/sm7150-xiaomi-tucana.dts

```diff
@@ -138,6 +225,42 @@
 	enable-gpios = <&tlmm 67 GPIO_ACTIVE_HIGH>;
 };
 
+&panel {
+	compatible = "xiaomi,f4-41-06-0a";
+};
+
+&pm6150l_flash {
+	status = "okay";
+
+	/* Both LEDs */
+	led-1 {
+		function = LED_FUNCTION_FLASH;
+		color = <LED_COLOR_ID_WHITE>;
+		led-sources = <1 2>;
+		led-max-microamp = <300000>;
+		flash-max-microamp = <2000000>;
+		flash-max-timeout-us = <1280000>;
+	};
+	/* Only first LED */
+	led-2 {
+		function = LED_FUNCTION_FLASH;
+		color = <LED_COLOR_ID_WHITE>;
+		led-sources = <1>;
+		led-max-microamp = <150000>;
+		flash-max-microamp = <1000000>;
+		flash-max-timeout-us = <1280000>;
+	};
+	/* Only second LED */
+	led-3 {
+		function = LED_FUNCTION_FLASH;
+		color = <LED_COLOR_ID_WHITE>;
```

Comment:
If multiple LEDs are of the same color, use different IDs in "function-enumerator" for each subnode (look at davinci's dts).

function-enumerator = <0>;